scene/resources/default_theme/default_theme.cpp

```diff
@@ -882,6 +882,10 @@ void fill_default_theme(Ref<Theme> &theme, const Ref<Font> &default_font, const
 
 	theme->set_icon("logo", "Icons", make_icon(logo_png));
 
+	// Visual Node Ports
+	theme->set_constant("port_grab_distance_horizontal", "GraphEdit", 150 * scale);
```

Comment:
why cant this be pixels, like everything else?

Reply:
Done. Moved the two grab_r variables into the constant.